lib/potassium/recipes/heroku.rb

```diff
@@ -47,31 +43,34 @@ def add_heroku
     template "../assets/bin/setup_heroku.erb", "bin/setup_heroku", force: true
     run "chmod a+x bin/setup_heroku"
 
-    if logged_in?
-      %w(staging production).each do |environment|
-        create_app_on_heroku(environment)
-      end
-      puts "Remember to connect the github repository to the new pipeline"
-      open_pipeline_command = "\e[33mheroku pipelines:open #{heroku_pipeline_name}\e[0m"
-      puts "run #{open_pipeline_command} to open the dashboard"
-    else
-      puts "You are not logged in into heroku"
-      login_command = "\e[33mheroku login\e[0m"
-      puts "Run #{login_command} and enter your credentials"
-      puts "You can install the heroku recipe again create the app in heroku"
-      install_command = "\e[33mpostassium install heroku --force\e[0m"
-      puts "Just run #{install_command}"
-    end
+    logged_in? ? create_apps : puts_not_logged_in_msg
 
     add_readme_header :deployment
   end
 
+  def create_apps
+    ENVIRONMENTS.each { |environment| create_app_on_heroku(environment) }
+    puts "Remember to connect the github repository to the new pipeline"
+    open_pipeline_command = "\e[33mheroku pipelines:open #{heroku_pipeline_name}\e[0m"
+    puts "run #{open_pipeline_command} to open the dashboard"
+  end
+
+  def puts_not_logged_in_msg
+    puts "You are not logged in into heroku"
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Rails/Output: Do not write to stdout. Use Rails's logger if you want to log.